tests/cli-self-update.rs

```diff
@@ -38,8 +37,8 @@ pub fn setup(f: &Fn(&Config)) {
 
         // An windows these tests mess with the user's PATH. Save
         // and restore them here to keep from trashing things.
-        let ref saved_path = get_path();
```

Comment:
This may be totally baseless, but could it be related to this uncommon use of `ref` here?